src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
+  const priceForDay = 40;
+  const count = days * priceForDay;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
+
+  return count - discount;
```

Comment:
It's better to calculate the total cost (`count`) after applying the discount. This way, you avoid unnecessary calculations if a discount is applied.